src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/DuplexClientBaseTests.cs

```diff
@@ -15,10 +13,45 @@ public static class DuplexClientBaseTests
 
     [Fact]
     [OuterLoop]
-    public static void DuplexClientBaseOfT_OverNetTcp_Call()
+    public static void DuplexClientBaseOfT_OverHttp_Call_Throws_InvalidOperation()
+    {
+        DuplexClientBase<IWcfDuplexService> duplexService = null;
+        Guid guid = Guid.NewGuid();
+
```

Comment:
Aside: this test failed for me running against our TFS CoreCLR/NETNative test environment.